src/homework07/Bankomat.java

```diff
@@ -0,0 +1,143 @@
+package homework07;
+
+public class Bankomat {
+
+    private int by20;
+    private int by50;
+    private int by100;
+
+    public String getTotalSumma() {
+
+        return "Total summa in ATM: " + (by20 * 20 + by50 * 50 + by100 * 100) +
+                "\nby20: " + by20 +
+                "\nby50: " + by50 +
+                "\nby100: " + by100;
+    }
+
+    public int restOfMoney() {
+        return by20 * 20 + by50 * 50 + by100 * 100;
+    }
+
+
+    public boolean load(int by20, int by50, int by100) {
```

Comment:
You create methods with return type but never use it. Two options: use void type or print some result of this methods.
Why it's important. In real projects we use automation check systems and they could send errors even in case such small issues.

Reply:
Thanks